.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+    - "3.8"
+
+install: pip install -r requirements.txt
+
+script:
+  - pylint department-app/tests
+  - coverage run department-app/tests
+after-success:
+  - coveralls
```

Comment:
Please, add an ability to send email notifications.

Reply:
done